lib/roll/app_builder.rb

```diff
@@ -79,11 +79,11 @@ def configure_generators
     end
 
     def configure_mailers_preview_path
-      config = <<-RUBY
+      config = <<-'RUBY'
 
 
   # Specific mailers path
-  config.action_mailer.preview_path = Rails.root + '/spec/mailers/previews'
```

Comment:
`Rails.root` isn't a string here. If we want to write it like this, we need to use `Rails.root.to_s`, which is just :poop:, isn't it ?